### ISSUE TITLE
Bug: The 'Running the operator locally' documentation might not work for all Windows VSCode Remote-Container Users

### ISSUE THREAD

**Issue by huangpf** [Azure/azure-service-operator#859]
**Describe the bug**
The 'Running the operator locally' documentation (link: 'https://github.com/Azure/azure-service-operator/blob/master/docs/development.md#running-the-operator-locally') might not work for all Windows VSCode Remote-Container users in that -- after the build and connection of the container, the 'Makefile' and several other files won't appear, so the dev-test process can't continue.

**To Reproduce**
1. Per instruction 'Clone the repository into the following folder <GOPATH>/src/github.com/Azure.' --- clone the repo into: 'C:\Go\bin\src\github.com\Azure\';
2. Open VS code and open the folder 'C:\Go\bin\src\github.com\Azure\.devcontainer';
3. Run 'build and remote-container'
4. The Makefile will not be in the final 'workspace' folder

**Expected behavior**
The Makefile should be in the final 'workspace' folder, and the dev-test continue;
Or the documentation shouldn't instruct Windows users to clone and use that path: 'C:\Go\bin\src\github.com\Azure\'.

**Screenshots**
![image](https://user-images.githubusercontent.com/3858524/78084256-88fa7900-736c-11ea-8840-19a162d58142.png)
![image](https://user-images.githubusercontent.com/3858524/78084540-4d13e380-736d-11ea-80e6-11f4cdb060f1.png)

**Additional context**
After cloning into this folder and opening it as the VSCode folder, issues resolved.
D:\git\asodev

In summary, either the documentation or the path loading/processing logic should be addressed.
